akka-docs/release-train-issue-template.md

```diff
@@ -18,6 +18,7 @@ Key links:
 
 - [ ] Check that open PRs and issues assigned to the milestone are reasonable
 - [ ] Update the version and change date in the LICENSE file.
+- [ ] Update the Akka Akka version in the samples to $VERSION$, otherwise the published zip files of the samples will have the old version.
```

Comment:
```suggestion
- [ ] Update the Akka version in the samples to $VERSION$, otherwise the published zip files of the samples will have the old version.
```